trpcultivate_phenoshare/src/Plugin/TripalImporter/TripalCultivatePhenoShareImporter.php

```diff
@@ -998,6 +1004,11 @@ public function processValidationMessages($failures) {
         'status' => 'todo',
         'details' => '',
       ],
+      'germplasm_name_exists' => [
+        'title' => 'Germplasm Name exists in the database',
```

Comment:
The file might have one or multiple germplasm
```suggestion
        'title' => 'Germplasm exist(s) in the database',
```